test/ci-cd/scripts/comfyui/run_in_cluster.sh

```diff
@@ -0,0 +1,189 @@
+STEP_ID=${1:-build-ci} # Default STEP_ID to 'build-ci' if not provided
+
+# --- Helpers ---
+step() { echo -e "\n==== [STEP] $* ====\n"; }
+info() { echo "[INFO] $*"; }
+warn() { echo "[WARN] $*" >&2; }
+
+# This function is triggered by the ERR trap or called directly.
+# It logs an error message and always exits 0.
+log_error() {
+  local exit_code=${1:-$?} # Use provided exit code or last command's
+  local message=${2:-"Script error on line ${BASH_LINENO[0]} (exit code: ${exit_code}) executing: ${BASH_COMMAND}"}
+
+  # Ensure we don't log successful exits when called from the ERR trap
+  if [ "${exit_code}" -eq 0 ] && [ "$#" -eq 0 ]; then
+    return
+  fi
+
+  >"${ERROR_FILE}" # Clear the error file
+  echo "- [${STEP_ID}] ${message}" >>"${ERROR_FILE}"
+  exit 0 # Always exit 0
+}
+trap 'log_error' ERR
+
+# This function runs on any script exit.
+# It cleans up the pod and ALWAYS exits 0.
+cleanup_on_exit() {
+  step "Cleanup"
+  info "Deleting pod: ${POD_NAME} (namespace: ${comfyui_kubernetes_namespace})"
+  kubectl delete pod "${POD_NAME}" -n "${comfyui_kubernetes_namespace}" --ignore-not-found=true >/dev/null 2>&1 || true
+
+  step "Final status"
+  if [ -s "${ERROR_FILE}" ]; then
+    warn "Test run had failures. See '${ERROR_FILE}' for details."
+    step "ERROR FILE CONTENTS"
+    cat "${ERROR_FILE}" || true
+  else
+    info "Test run completed successfully."
+  fi
+
+  # ALWAYS exit 0 per requirement
+  info "Exiting with status 0 to ensure build step passes."
+  exit 0
+}
+trap cleanup_on_exit EXIT
+
+# ------------------------------------------------------------
+# Start client pod
+# ------------------------------------------------------------
+step "Create client pod '${POD_NAME}' in '${comfyui_kubernetes_namespace}'"
+export SA_NAME=$(kubectl get serviceaccount -n ${comfyui_kubernetes_namespace} -o custom-columns=NAME:.metadata.name --no-headers | grep -v "^default$")
+kubectl run "${POD_NAME}" \
+  --image=alpine:latest \
+  --restart=Never \
+  -n "${comfyui_kubernetes_namespace}" \
+  --overrides="{ \"spec\": { \"serviceAccountName\": \"${SA_NAME}\" } }" \
+  --command -- sh -c "apk add --no-cache bash curl jq >/dev/null && echo 'Pod is ready. Waiting...' && sleep 3600"
+
+step "Wait for pod to be Ready"
+kubectl wait --for=condition=Ready "pod/${POD_NAME}" -n "${comfyui_kubernetes_namespace}" --timeout=3600s
+
+# ------------------------------------------------------------
+# Copy test assets
+# ------------------------------------------------------------
+step "Copy test assets into pod"
+info "Copying comfyui_prompt_test.sh and workflows directory"
+TEMP_DIR="${TEST_WORKFLOW_DIR}/tmp"
+mkdir -p "${TEMP_DIR}"
+cp -r "${TEST_WORKFLOW_DIR}/workflows/"* "${TEMP_DIR}"
+for file in "${TEMP_DIR}"/*; do
+    if [[ -f "$file" ]]; then
+        envsubst < "$file" | sponge "$file"
+    fi
+done
+kubectl cp "${TEST_WORKFLOW_DIR}/images" "${comfyui_kubernetes_namespace}/${POD_NAME}:/tmp/images"
```

Comment:
I don't see an `images` directory in the `test/ci-cd/scripts/comfyui` directory.